### PR TITLE
chore(appflowy_flutter): update to Flutter 3.7.5

### LINKED ISSUE #1954: [FR] Upgrade to Flutter 3.7.5
### Description

Upgrade AppFlowy and its packages to Flutter 3.7.5.

- [x] fork `flutter_math_fork` to upgrade its depending `flutter_svg` version to `^2.0.2`.
- [ ] upgrade `flutter_svg` for `flowy_infra`.
- [ ] remove AppFlowy Editor from AppFlowy repo.
- [x] #1958

### Impact

Newer versions of Flutter may include bug fixes, security fixes, performance improvements, and new features

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by hyj1204**
- [X] run `flutter pub upgrade --major-versions` and update dependencies version
 - [X] Due to `color` is deprecated in `flutter_svg` package, update it with 
 ```
colorFilter:
            color != null ? ColorFilter.mode(color!, BlendMode.srcIn) : null,
```
- [X] Delete unnecessary code, deprecated field and formatting code

This PR followed the work in #1990 
Fixes #1954

### CODE REVIEW

File: .github/workflows/appflowy_editor_test.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+name: AppFlowyEditor test
```
Comment (LucasXu0) [inline .github/workflows/appflowy_editor_test.yml:1]: Don't add this file back. We have moved it to the `appflowy_editor` repo

**Comment by LucasXu0** [comment]
@hyj1204  The `integrate_test` failed.

**Comment by LucasXu0** [comment]
Please update this document too. https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/environment-setup

File: .github/workflows/appflowy_editor_test.yml (hunk 1 above)
Comment (hyj1204) [reply .github/workflows/appflowy_editor_test.yml]: Deleted

**Comment by hyj1204** [comment]
> Please update this document too. https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/environment-setup

Done. 
https://github.com/AppFlowy-IO/AppFlowy-Docs/pull/57

**Comment by LucasXu0** [comment]
LGTM. Will merge it after releasing v0.1.1.

**Pull request merged**